### DISCUSSION TITLE
AcrylicEverywhere's AccentBlur Overrider brought back to DWMBlurGlass

### DISCUSSION THREAD

**Discussion by kfh83** [Maplespe/DWMBlurGlass#225] [Ideas]
As title says. AcrylicEverywhere currently supports overriding AccentBlur with it's own blur:
![image](https://github.com/Maplespe/DWMBlurGlass/assets/85626296/af71eab9-f89b-40a6-b6ae-2393b035a1a4)
I think it would probably be a good idea to bring this back to DWMBlurGlass as a config.ini switch or experimental option in the GUI or something, especially since you can very clearly notice DComp's biggest shortcoming (30 FPS backdrop refresh), but it would still be useful for those who wish for reflections and accurate colorization on the taskbar/start menu.

**Comment by ALTaleX531** [comment]
Don't worry, I'll be porting most of the AcrylicEverywhere features to DwmBlurGlass, which will definitely have a separate switch. However, the low frame rate refresh of the backdrop occurs throughout the composition engine, which is an intentional optimization behavior, even with our custom blur method. I've thought about implementing it in a BigMusle way, but considering that the composition engine itself has optimizations such as caching and occlusion, as well as a wide variety of Windows versions, it's just asking for trouble.

**Reply by kfh83** [reply]
Is there _really_ no way to make DComp refresh higher than 30 FPS?

**Reply by ALTaleX531** [reply]
Unfortunately, there is currently no way to remove this restriction.

**Reply by kfh83** [reply]
Unrelated, but do you have anything like discord or similar?

**Reply by ALTaleX531** [reply]
Before you ask, I don't have a Discord account yet, I just created one for this purpose and you can now reach me at ``.
